tutorials/advanced_features/Local addressability with DMM.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<p>You can use <code>assert</code> instead of <code>print</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Okay I hesitated between both :) </p>